airflow/providers/slack/hooks/slack.py

```diff
@@ -39,6 +39,8 @@
 
 def _ensure_prefixes(conn_type):
     """
+    Deprecated.
+
     Remove when provider min airflow version >= 2.5.0 since this is handled by
     provider manager from that version.
     """
```

Comment:
Better to convert this to a TODO comment, since this docstring is not shown in the documentation.